public/app/plugins/datasource/prometheus/language_provider.ts

```diff
@@ -525,17 +560,17 @@ export default class PromQlLanguageProvider extends LanguageProvider {
    */
   fetchSeriesValuesWithMatch = async (name: string, match?: string): Promise<string[]> => {
     const interpolatedName = name ? this.datasource.interpolateString(name) : null;
-    const range = this.datasource.getTimeRangeParams();
+    const range = this.datasource.getQuantizedTimeRangeParams();
     const urlParams = {
       ...range,
       ...(match && { 'match[]': match }),
     };
 
     const cacheParams = new URLSearchParams({
-      'match[]': interpolatedName ?? '',
```

Comment:
so I think the current values are not correct, we're not cacheing on the match string so this could potentially be  a bug on main